src/shared/logger/winstonToolkitLogger.ts

```diff
@@ -93,6 +93,11 @@ export class WinstonToolkitLogger implements Logger, vscode.Disposable {
     }
 
     private writeToLogs(level: LogLevel, message: string | Error, ...meta: any[]): void {
+        // Don't write new lines after a log message, makes it look strange
+        if (typeof(message) == "string") {
```

Comment:
Since it's just removing trailing whitespace and newlines, it shouldn't affect other statements in a significant way. Only logging statements that were trying to create an empty line after their message would be affected.